helm/charts/determined/templates/master-config.yaml

```diff
@@ -152,7 +152,7 @@ stringData:
 
     resource_manager:
       type: "kubernetes"
-      namespace: {{ .Release.Namespace }}
+      default_namespace: {{ .Release.Namespace }}
```

Comment:
note: I know the feature isn't ready for release yet, but until it is I think we should (1) either make namespace coeexist with default_namespace (so make this change backwards compatible) or (2) wait until you're ready to announce & update docs at the same time
personally I lean towards (1). as it stands, if this change makes it into a release i think it would break the helm charts/configs of existing customers

Reply:
done!